arch/sim/src/sim/up_initialize.c

```diff
@@ -31,12 +31,30 @@
 #include <nuttx/spi/spi_flash.h>
 #include <nuttx/spi/qspi_flash.h>
 
+#include <stdlib.h>
+
 #include "up_internal.h"
 
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
 
+#ifndef CONFIG_DISABLE_ENVIRON
+static void up_init_cmdline(void)
+{
+  char cmdline[ARG_MAX] = "";
+  int i;
+
+  for (i = 1; i < g_argc; i++)
+    {
+      strcat(cmdline, g_argv[i]);
```

Comment:
It could be a good idea to use strncat and ARG_MAX to make it a little bit safer

Reply:
Done.